client/components/translator-invite/index.jsx

```diff
@@ -28,50 +28,110 @@ export class TranslatorInvite extends Component {
 	recordClick = () =>
 		this.props.recordTracksEvent( 'calypso_translator_invitation', {
 			language: this.props.localizedLanguageNames[ this.props.locale ].en,
+			location: this.props.path,
 		} );
 
+	renderIcon() {
+		return <MaterialIcon icon="emoji_language" />;
+	}
+
+	renderHeader() {
+		const { translate } = this.props;
+		return (
+			<p className="card-heading card-heading-20">{ translate( 'Translate WordPress.com' ) }</p>
+		);
+	}
+
+	renderLearnMoreLink = ( isInline = false ) => {
+		const { locale, translate } = this.props;
+		const link = (
+			<a
+				className="language-picker__modal-incomplete-locale-notice-learn-more"
+				href={ `https://translate.wordpress.com/projects/wpcom/${ locale }/default/` }
+				target="_blank"
+				rel="noopener noreferrer"
+				onClick={ this.recordClick }
+			>
+				{ translate( 'Learn more' ) }
+			</a>
+		);
+
+		if ( ! isInline ) {
+			return <div>{ link }</div>;
+		}
+
+		return link;
+	};
+
+	renderIconAndHeader = () => {
+		const { path, translate } = this.props;
+
+		switch ( path ) {
+			case '/home':
+				return (
+					<>
+						<MaterialIcon icon="emoji_language" />
+						<p className="card-heading card-heading-20">
```

Comment:
Using the class names from [the CardHeading component](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/card-heading/index.jsx#L12) will only provide the expected styling if the `CardHeading` is used elsewhere in the current user session so that it's stylesheet gets loaded, which may not always be the case.